Tradeview

```diff
@@ -1,138 +0,0 @@
-      "Principal": {
-        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
-      },
-      "Action": [
-        "s3:GetObject",
-        "s3:PutObject"
-      ],
-      "Resource": "arn:aws:s3:::cmemtl/CME_DATA/*"
-    }
-  ]
-}curl --request GET \
-  -u ORGANIZATION_ID:API_KEY \
-  --url https://app.moderntreasury.com/api/ping
-  {
-  "Id": "PolicyDatamineGenerated",
-  "Statement": [
-    {
-      "Sid": "1",
-      "Effect": "Allow",
-      "Principal": {
-        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
-      },
-      "Action": "s3:ListBucket",
-      "Resource": "arn:aws:s3:::cmemtl"
-    },
-    {
-      "Sid": "2",
-      "Effect": "Allow",
-      "Principal": {
-        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
-      },
-      "Action": [
-        "s3:GetObject",
-        "s3:PutObject"
-      ],
-      "Resource": "arn:aws:s3:::cmemtl/CME_DATA/*"
-    }
-  ]
-}
-curl -H "Authorization: Bearer 
-285ccde5b212309185d8088775b7b506-7dc53404dac4cd0441b7cabd91700e35
-
-https://api-fxtrade.oanda.com/v3/accounts
-{
-    # 
-    # The Transaction’s Identifier.
-    # 
-    id : (TransactionID),
-
-    # 
-    # The date/time when the Transaction was created.
-    # 
-    time : (DateTime),
-
-    # 
-    # The ID of the user that initiated the creation of the Transaction.
-    # 
-    userID : (integer),
-
-    # 
-    # The ID of the Account the Transaction was created for.
-    # 
-    accountID : (AccountID),
-
-    # 
-    # The ID of the “batch” that the Transaction belongs to. Transactions in
-    # the same batch are applied to the Account simultaneously.
-    # 
-    batchID : (TransactionID),
-
-    # 
-    # The Request ID of the request which generated the transaction.
-    # 
-    requestID : (RequestID),
-
-    # 
-    # The Type of the Transaction. Always set to “CLIENT_CONFIGURE” in a
-    # ClientConfigureTransaction.
-    # 
-    type : (TransactionType, default=CLIENT_CONFIGURE),
-
-    # 
-    # The client-provided alias for the Account.
-    # 
-    alias : (string),
-
-    # 
-    # The margin rate override for the Account.
-    # 
-    marginRate : (DecimalNumber)
-    curl ---request GET \
-  -u ORGANIZATION_ID:API_KEY \ 
-  --url https://app.moderntreasury.com/api/api_keys/current
-  curl -H "Authorization: Bearer 
-285ccde5b212309185d8088775b7b506-7dc53404dac4cd0441b7cabd91700e35
-
-https://api-fxtrade.oanda.com/v3/accounts
-{
-  "Id": "PolicyDatamineGenerated",
-  "Statement": [
-    {
-      "Sid": "1",
-      "Effect": "Allow",
-      "Principal": {
-        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
-      },
-      "Action": "s3:ListBucket",
-      "Resource": "arn:aws:s3:::cmemtl"
-    },
-    {
-      "Sid": "2",
-      "Effect": "Allow",
-      "Principal": {
-        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
-      },
-      "Action": [
-        "s3:GetObject",
-        "s3:PutObject"
-      ],
-      "Resource": "arn:aws:s3:::cmemtl/CME_DATA/*"
-    }
-  ]
-}
```

Comment:
curl --request GET \
  -u ORGANIZATION_ID:API_KEY \
  --url https://app.moderntreasury.com/api/ping{
  "Id": "PolicyDatamineGenerated",
  "Statement": [
    {
      "Sid": "1",
      "Effect": "Allow",
      "Principal": {
        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
      },
      "Action": "s3:ListBucket",
      "Resource": "arn:aws:s3:::cmemtl"
    },
    {
      "Sid": "2",
      "Effect": "Allow",
      "Principal": {
        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
      },
      "Action": [
        "s3:GetObject",
        "s3:PutObject"
      ],
      "Resource": "arn:aws:s3:::cmemtl/CME_DATA/*"
    }
  ]
curl ---request GET \
  -u ORGANIZATION_ID:API_KEY \ 
  --url https://app.moderntreasury.com/api/api_keys/current{
  "Id": "PolicyDatamineGenerated",
  "Statement": [
    {
      "Sid": "1",
      "Effect": "Allow",
      "Principal": {
        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
      },
      "Action": "s3:ListBucket",
      "Resource": "arn:aws:s3:::cmemtl"
    },
    {
      "Sid": "2",
      "Effect": "Allow",
      "Principal": {
        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
      },
      "Action": [
        "s3:GetObject",
        "s3:PutObject"
      ],
      "Resource": "arn:aws:s3:::cmemtl/CME_DATA/*"
    }
  ]
curl -H "Authorization: Bearer 
285ccde5b212309185d8088775b7b506-7dc53404dac4cd0441b7cabd91700e35

https://api-fxtrade.oanda.com/v3/accounts{
  "Id": "PolicyDatamineGenerated",
  "Statement": [
    {
      "Sid": "1",
      "Effect": "Allow",
      "Principal": {
        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
      },
      "Action": "s3:ListBucket",
      "Resource": "arn:aws:s3:::cmemtl"
    },
    {
      "Sid": "2",
      "Effect": "Allow",
      "Principal": {
        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
      },
      "Action": [
        "s3:GetObject",
        "s3:PutObject"
      ],
      "Resource": "arn:aws:s3:::cmemtl/CME_DATA/*"
    }
  ]
}
[Api.Documentation (1).zip](https://github.com/TRUST-OF-MECCA-EXPRESS-INC/example-api-js/files/7623206/Api.Documentation.1.zip)
curl -H "Authorization: Bearer 
285ccde5b212309185d8088775b7b506-7dc53404dac4cd0441b7cabd91700e35

https://api-fxtrade.oanda.com/v3/accountscurl --request GET \
  -u ORGANIZATION_ID:API_KEY \
  --url https://app.moderntreasury.com/api/ping{
  "Id": "PolicyDatamineGenerated",
  "Statement": [
    {
      "Sid": "1",
      "Effect": "Allow",
      "Principal": {
        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
      },
      "Action": "s3:ListBucket",
      "Resource": "arn:aws:s3:::cmemtl"
    },
    {
      "Sid": "2",
      "Effect": "Allow",
      "Principal": {
        "AWS": "arn:aws:iam::920069880949:user/ticksmith"
      },
      "Action": [
        "s3:GetObject",
        "s3:PutObject"
      ],
      "Resource": "arn:aws:s3:::cmemtl/CME_DATA/*"
    }
  ]
curl -H "Authorization: Bearer 
285ccde5b212309185d8088775b7b506-7dc53404dac4cd0441b7cabd91700e35

https://api-fxtrade.oanda.com/v3/accountscurl ---request GET \
  -u ORGANIZATION_ID:API_KEY \ 
  --url https://app.moderntreasury.com/api/api_keys/current